vtex/vtex.js
# Feat/contextual pipeline

#### What is the purpose of this pull request?

Gets updated version of orderForm to avoid inconsistencies in cart and checkout.

#### What problem is this solving?

Avoids outdated orderForm information in cart and checkout

#### How should this be manually tested?
On the `vtex.js` project folder:
`yarn install`
`yarn grunt dev`

On the `vcs.checkout-ui` project folder:
`yarn install`
`yarn grunt --link=vtex.js`

Then, open [here]() and open the network tab in dev tools. Go to the cart page and check the XHR requests to see if the query string is added to the orderForm request.

#### Screenshots or example usage
<img width="1438" alt="Screen Shot 2021-04-16 at 3 59 41 PM" src="https://user-images.githubusercontent.com/49081447/115071549-c32dae80-9ecc-11eb-8018-ade080d1b635.png">


#### Types of changes
